### ISSUE TITLE
Crashes when trying to edit custom, downloadable versions

### ISSUE THREAD

**Issue by Mai-Lapyst** [eumario/godot-manager#53]
**Describe the bug**
Crashes when trying to edit custom versions.

**To Reproduce**
1. Go to the version manager
2. Click on "Download from Website" button
3. Click on the pencil button on the left when the inputs on the right are empty (no version selected)

**Expected behavior**
Not crashing

**Screenshots**
No screenshots but a log:
```
Unhandled Exception:
System.UriFormatException: Invalid URI: The URI is empty.
  at System.Uri.CreateThis (System.String uri, System.Boolean dontEscape, System.UriKind uriKind) [0x0007b] in <7d230238b7e0493bad02e7b5f5cb8aa8>:0 
  at System.Uri..ctor (System.String uriString) [0x00014] in <7d230238b7e0493bad02e7b5f5cb8aa8>:0 
  at ManageCustomDownloads.UpdateStruct () [0x0009f] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0 
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__7_0 (System.Object state) [0x00000] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at Godot.GodotSynchronizationContext.ExecutePendingContinuations () [0x0001d] in <d0b7d6b6d74e4318be3dfcff5c3b7c05>:0 
  at Godot.GodotTaskScheduler.Activate () [0x0000c] in <d0b7d6b6d74e4318be3dfcff5c3b7c05>:0 
  at (wrapper native-to-managed) Godot.GodotTaskScheduler.Activate(Godot.GodotTaskScheduler,System.Exception&)

Unhandled Exception:
System.UriFormatException: Invalid URI: The URI is empty.
  at System.Uri.CreateThis (System.String uri, System.Boolean dontEscape, System.UriKind uriKind) [0x0007b] in <7d230238b7e0493bad02e7b5f5cb8aa8>:0 
  at System.Uri..ctor (System.String uriString) [0x00014] in <7d230238b7e0493bad02e7b5f5cb8aa8>:0 
  at ManageCustomDownloads.UpdateStruct () [0x0009f] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0 
  at System.Runtime.CompilerServices.AsyncMethodBuilderCore+<>c.<ThrowAsync>b__7_0 (System.Object state) [0x00000] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at Godot.GodotSynchronizationContext.ExecutePendingContinuations () [0x0001d] in <d0b7d6b6d74e4318be3dfcff5c3b7c05>:0 
  at Godot.GodotTaskScheduler.Activate () [0x0000c] in <d0b7d6b6d74e4318be3dfcff5c3b7c05>:0 
  at (wrapper native-to-managed) Godot.GodotTaskScheduler.Activate(Godot.GodotTaskScheduler,System.Exception&)
```

**Complete the Following:**
 - OS: Linux
 - Version 0.2.1

**Comment by eumario** [comment]
Pencil (Edit) and Dash (Remove, need to change to Trash Can Icon) need to be disabled when user has not selected anything.  Will be fixed in 0.3.0.

**Comment by eumario** [comment]
Commmit [e0961eb](https://github.com/eumario/godot-manager/commit/e0961eb05c0aacfe8dfe112194e3a72031de6fff) fixes bug.  Currently will check to see if an Entry is selected, before allowing the edit/remove functionality to execute.  Will warn user that an entry needs to be selected in order to edit it.
